### PR TITLE
send email for autorebuilds to submitter from original task

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-8395

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/plugins/exit_sendmail.py [hunk 1]
```diff
@@ -343,7 +352,10 @@ def _get_email_from_koji_obj(self, obj):
             return '@'.join([obj['name'], self.email_domain])
 
     def _get_koji_submitter(self):
-        koji_task_owner = get_koji_task_owner(self.session, self.koji_task_id)
+        if self.original_koji_task_id:
+            koji_task_owner = get_koji_task_owner(self.session, self.original_koji_task_id)
+        else:
+            koji_task_owner = get_koji_task_owner(self.session, self.koji_task_id)
```
Comment (twaugh) [inline atomic_reactor/plugins/exit_sendmail.py]: Minor: could be simplified with
```
koji_task_owner = get_koji_task_owner(self.session, self.original_koji_task_id or self.koji_task_id)
```

File: atomic_reactor/plugins/exit_sendmail.py [hunk 2]
```diff
@@ -173,6 +173,15 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
+        try:
+            metadata = get_build_json().get("metadata", {})
+            self.original_koji_task_id = int(metadata['labels']['original-koji-task-id'])
+        except Exception:
```
Comment (athos-ribeiro) [inline atomic_reactor/plugins/exit_sendmail.py]: What other exceptions are we expecting here other than TypeError or KeyError? Could we be explicit on those here?

File: atomic_reactor/plugins/exit_sendmail.py (hunk 1 above)
Comment (rcerven) [reply atomic_reactor/plugins/exit_sendmail.py]: ok

File: atomic_reactor/plugins/exit_sendmail.py (hunk 2 above)
Comment (rcerven) [reply atomic_reactor/plugins/exit_sendmail.py]: using the same as in getting koji-task-id above, we don't really care that much in this case how it fails, we either get value or not

**Comment by rcerven** [comment]
rn updated: send email for autorebuilds to submitter from original task

**Pull request merged**